Kernel/linker.ld

```diff
@@ -101,6 +101,7 @@ SECTIONS
     {
         start_of_kernel_ksyms = .;
         *(.kernel_symbols)
+        *(.got*)
```

Comment:
Making a note here so we don't merge it like this. This shouldn't be in `.ksyms`. :)